CMakeLists.txt

```diff
@@ -426,6 +428,27 @@ elseif(UMF_BUILD_CUDA_PROVIDER)
     message(STATUS "CUDA_INCLUDE_DIRS = ${CUDA_INCLUDE_DIRS}")
 endif()
 
+if(WINDOWS AND UMF_USE_DEBUG_POSTFIX)
+    # Build debug umf library with the d suffix that is compiled with /MDd so
+    # users can link against it in debug builds.
+    set(CMAKE_DEBUG_POSTFIX d)
+
+    add_custom_target(
+        umfd ALL
+        COMMAND ${CMAKE_COMMAND} --build ${CMAKE_BINARY_DIR} --target umf
+                --config Debug
+        COMMENT "Building debug umf library with the d suffix")
+
+    # Copy built UMF libraries to the Release build subdirectory
+    add_custom_command(
+        TARGET umfd
+        COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_BINARY_DIR}/bin/Debug/umfd.dll
+                ${CMAKE_BINARY_DIR}/bin/Release/umfd.dll
+        COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_BINARY_DIR}/lib/Debug/umfd.lib
+                ${CMAKE_BINARY_DIR}/lib/Release/umfd.lib
+        COMMENT "Copying debug libraries to the Release build directory")
```

Comment:
This breaks when using Ninja generator (I assume also Unix Makefiles which is what internal builds use).

Reply:
Changes made in https://github.com/oneapi-src/unified-memory-framework/pull/1238